src/components/About.js

```diff
@@ -0,0 +1,58 @@
+import { Link } from "react-router-dom";
+
+
+const About = () => {
+    return ( 
+        <>
+
+  
+    <div className="container mt-5 lead">
+  <div className="row">
+  <img src="img/footer/welcome.png" height="200px" alt="pizza"/>
```

Comment:
Could you move this to About folder in images directory? Makes more sense.

Reply:
done

Reply:
👍